### PR TITLE
1905-12-04

### PR DESCRIPTION
**Pull request by Kf15b**
Just need to finish adding advertisements.

### CODE REVIEW

File: 1905-12-04.xml [hunk 1]
```diff
@@ -0,0 +1,430 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="https://raw.githubusercontent.com/dig-eg-gaz/resources/master/out/egSchema.rnc" type="application/relax-ng-compact-syntax"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title level="m" type="main">Digital Egyptian Gazette</title>
+                <title level="m" type="sub">An encoded transcription</title>
+                <editor><!-- Katherine Ferguson--></editor>
```
Comment (znr13) [inline 1905-12-04.xml]: Remove comment brackets

**Comment by Kf15b** [comment]
All Ads have been added and page 3 is done as well. Just need to work on page two, four,five,six,seven, and eight.

File: 1905-12-04.xml [hunk 2]
```diff
@@ -0,0 +1,1267 @@
...
+                </edition>
+            </editionStmt>
+            <publicationStmt>
+                <publisher>FSU University Libraries</publisher>
+                <pubPlace>Tallahassee, FL</pubPlace>
+            </publicationStmt>
+            <sourceDesc>
+                <bibl>
+                    <title>The Egyptian Gazette</title>
+                    <date when="1905-01-04">December 04, 1905<!-- 1905-12-04 --></date>
```
Comment (znr13) [inline 1905-12-04.xml]: Correct date.

**Comment by znr13** [comment]
@Kf15b , correct the date issue and keep up the hard work.

**Pull request merged**